keyboards/412_64/readme.md

```diff
@@ -0,0 +1,15 @@
+# 412-64
+
+![412-64](image_here)
+
+A compact 60% keyboard designed by Fate Everywhere and sold on a limited basis. Born out of the desire for a Planck with a numpad for technical work.
+
+Keyboard Maintainer: [Fate Everywhere](https://github.com/fateeverywhere)  
+Hardware Supported: 4x16 T&E Prototype, 412-64 Mk. 0, Mk. 1.  
+Hardware Availability: Highly limited, contact /u/FateEverywhere on reddit for availability.
```

Comment:
If this is the case, then we'd like this to be moved to the "handwired" folder.  

Even if the board isn't handwired, we put very small runs, personal PCBs and handwired boards in that folder.

If you or whomever is planning on running a group buy for this very soon, then it's fine where it is.  Otherwise, please move it.

Reply:
I do plan on running a group buy (and eventually perhaps opening a store) for the boards, but if it's inappropriate to drop the firmware here at this point I'll retract the pull request.